Source/ACE/Entity/Container.cs

```diff
@@ -51,14 +52,54 @@ public Container(AceObject aceObject, ObjectGuid guid)
         {
         }
 
+        public virtual void ContainerPlacement(WorldObject inventoryItem)
```

Comment:
I'm not a really big fan of this function name.  What does ContainerPlacement mean?  I would assume it returns the container placement of the item, but that's clearly not it, it removes an inventory item, then does some other stuff. Perhaps RemoveItemFromInventory?  Or would that be improper for some reason?